app/src/main/java/com/getdefault/task9/BindingAdapter.kt

```diff
@@ -0,0 +1,32 @@
+package com.getdefault.task9
+
+import android.widget.TextView
+import androidx.databinding.BindingAdapter
+import androidx.recyclerview.widget.RecyclerView
+import org.w3c.dom.Text
+
+@BindingAdapter("personData")
+fun bindRecyclerview(rv: RecyclerView, list: ArrayList<Person>) {
+    val adapter = rv.adapter as PersonAdapter
```

Comment:
@saurav-kalsoor-default will throw NPE if no adapter is set

Reply:
Ok sir, I'll add checks.